drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-api/src/main/java/org/drools/workbench/screens/scenariosimulation/model/FactMapping.java

```diff
@@ -67,7 +67,7 @@ public FactMapping() {
     }
 
     public FactMapping(FactIdentifier factIdentifier, ExpressionIdentifier expressionIdentifier) {
-        this(expressionIdentifier.getName(), factIdentifier, expressionIdentifier);
+        this(factIdentifier.getName(), factIdentifier, expressionIdentifier);
```

Comment:
This change could potentially have a big impact: did you test at runtime everything client-side is still working?

Reply:
This constructor has been used only for testing purpose that's why nobody found the bug before (and there is no impact on UI too). We can refactor the test and remove this code but for now at least is correct :)